java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -1769,16 +1769,16 @@ public boolean contains(Scalar needle) {
   }
 
   /**
-   * Returns a new ColumnVector of {@link DType#BOOL8} elements containing true if the corresponding
-   * entry in haystack is contained in needles and false if it is not. The caller will be responsible
-   * for the lifecycle of the new vector.
+   * Returns a new ColumnVector of {@link DType#BOOL8} elements containing true if the 
+   * corresponding entry in needles is contained in this column and false if it is not.
+   * The caller is responsible for the lifecycle of the new vector.
```

Comment:
By leaving out the "haystack," the analogy is now only half here, and some may be confused by what "needles" represents.